_install-and-configure/configuring-opensearch/index-settings.md

```diff
@@ -109,6 +109,8 @@ For `zstd`, `zstd_no_dict`, `qat_lz4`, and `qat_deflate`, you can specify the co
 
 - `index.check_pending_flush.enabled` (Boolean): This setting controls the Apache Lucene `checkPendingFlushOnUpdate` index writer setting, which specifies whether an indexing thread should check for pending flushes on an update in order to flush indexing buffers to disk. Default is `true`.
 
+- `index.use_compound_file` (Boolean): This setting controls the Apache Lucene `useCompoundFile` index writer settings, which specifies whether index segment files will be packaged into a compound file. Default is `true`
```

Comment:
```suggestion
- `index.use_compound_file` (Boolean): This setting controls the Apache Lucene `useCompoundFile` index writer settings, which specifies whether newly written segment files will be packed into a compound file. Default is `true`.
```

@samuel-oci What do you think? The original text could maybe have been misinterpreted as all segment files would be written in a compound file format. I took the wording from the [Lucene docs](https://lucene.apache.org/core/9_8_0/core/org/apache/lucene/index/LiveIndexWriterConfig.html#setUseCompoundFile(boolean)).

Reply:
It can always change after merges, but I assume if when you read "newly written" as something that can also be triggered via merges it will be more accurate :)
Sounds good, signed off